mono/mini/mini-exceptions.c

```diff
@@ -2990,12 +2990,11 @@ mono_handle_native_crash (const char *signal, void *ctx, MONO_SIG_HANDLER_INFO_T
 			if (!dump_for_merp) {
 #ifdef DISABLE_STRUCTURED_CRASH
```

Comment:
Consider compile-if instead  of preprocessor-if. It makes for better coverage when building other  platforms. i.e. less easy to break what you don't  build. ifdef is overrated and overused.

Reply:
I usually cppcheck and that does a good job of checking that the code behind all ifdefs are legal

Reply:
And this is partially why I'm doing what I do here with setting `leave = TRUE`, so the code below is enabled or disabled based on a compile-if.